assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/jgroups/che-tcp.xml

```diff
@@ -0,0 +1,54 @@
+<!--
+
+    Copyright (c) 2012-2018 Red Hat, Inc.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<config xmlns="urn:org:jgroups"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/JGroups-3.1.xsd">
+  <TCP
+    bind_addr="match-interface:eth0"
+    bind_port="${TCP_PORT:7800}"
+    recv_buf_size="5M"
+    send_buf_size="1M"
+    max_bundle_size="64K"
+    enable_diagnostics="true"
+    thread_naming_pattern="cl"
+
+    thread_pool.min_threads="0"
+    thread_pool.max_threads="500"
+    thread_pool.keep_alive_time="30000" />
+
+  <kubernetes.KUBE_PING
+    namespace="${OPENSHIFT_KUBE_PING_NAMESPACE:eclipse-che}"
```

Comment:
As far as I understand it is used for Kubernetes and OpenShift infrastructures. So let's rename env var to `KUBERNETES_KUBE_PING_NAMESPACE`, or maybe even `CHE_INFRA_KUBERNETES_KUBE__PING__NAMESPACE` to make it clear that it is infrastructure specific configuration property.

Reply:
I think we can remove it here at all. But we still should pass it to container as is:
See
https://github.com/jgroups-extras/jgroups-kubernetes/blob/0.9.3/kube/src/main/java/org/jgroups/ping/kube/KubePing.java#L96
+
https://github.com/jgroups-extras/jgroups-kubernetes/blob/0.9.3/kube/src/main/java/org/jgroups/ping/kube/KubePing.java#L131
I will try to remove it from here.

Reply:
Up to you, but I'd define our own documented property into che.properties and che.env.

Reply:
For me it seems we can't use own prop-s here.

Reply:
Can you describe why we can't? Is not it just name of environment property that should be respected?

Reply:
Checked the jgroups code. We can use custom system property. But env.variable should be named as in the links above.